Gemfile

```diff
@@ -82,7 +82,7 @@ group :jekyll_optional_dependencies do
 
   # Windows and JRuby does not include zoneinfo files, so bundle the tzinfo-data gem
   # and associated library
-  install_if -> { RUBY_PLATFORM =~ %r!mingw|mswin|java! } do
+  platform :jruby, :mswin, :mingw, :x64_mingw do
```

Comment:
```suggestion
  platforms :jruby, :mswin, :mingw, :x64_mingw do
```

Reply:
Is it platform or platforms?

Reply:
I just copied it from the section above in Gemfile (line 77). It seems both are used widely, but the manual page lists it as `platforms`.

Reply:
Pushed another commit